swap_meet/decor.py

```diff
@@ -1,2 +1,17 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+# from item import Item
+class Decor(Item):
+    def __init__(self, condition=0, category="Decor", age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+    def __str__(self):
+        return "Something to decorate your space."
+
+
+# # attempt to user super():
+# class Decor(Item):
+#     def __init__(self, condition=0, category="Decor", age = 0):
+#         super().__init__(self, condition=0, category="", age = 0)
```

Comment:
I'm guessing that this section is commented out because it wasn't working.  The reason why this didn't work is that the function call here to `__init__` is doing 2 things wrong - first, it's passing self explicitly, which will throw an error.  Second, the parameters are being passed as keyword arguments, which is fine, but the value is being set to the default values, rather than using the values that are being passed in to the Decor `__init__`.  Using keyword arguments here is fine, but the value needs to be the argument that's passed into the function (which in this case looks the same as the keyword, which results in code that looks a bit confusing):
```suggestion
#         super().__init__(condition=condition, category=category, age = age)
```